### DISCUSSION TITLE
News and Interests (Weather) icon on the taskbar if possible?

### DISCUSSION THREAD

**Discussion by novatiberium** [valinet/ExplorerPatcher#419] [Ideas]
In Windows 11, Microsoft has made "News and Interests" among with the weather icon showing the current conditions on the taskbar - short lived. Unless you have Windows 10 onwards.

I hate GitHub for them erasing what I just wrote, because I just selected a category and I'm new, but anyways - I know you guys programmed the Cortana button to be the Widgets feature, but while similar it doesn't offer the weather on the taskbar.

Is it possible to backport the Weather current conditions from Windows 10 into ExplorerPatcher? And while on it - maybe we could also backport the Windows 10 News and Interests (which is smaller than 11's) into there as well?

**Comment by valinet** [comment]
Yeah, I have looked into it a while ago, someone asked something similar; I don't remember what thread that was, it's somewhere in here though. I have looked into it, they seem to have removed the code entirely from Explorer, so this kind of patch is harder. Not impossible, but harder. I compared the `explorer.exe` from 19041 to the one in 22000, and the newer one just does not have a lot of things related to news and interests... Personally, I don't understand it either, why develop a feature just to nuke it 1 release after, but questionable business decisions seem to be the norm at Microsoft nowadays. The feed was pure garbage, but indeed, it would have been nice to quickly be able to glance at the weather without too much fuss. Of course, someone can reimplement it from scratch, it's not that hard, the taskbar supports custom toolbars since forever, someone did an Everything toolbar, for example, so this is totally doable, and easier I'd say to start from scratch, something similar to how I implemented the Windows 10 window switcher.

**Reply by abcpop** [reply]
I think they replace   "News and Interests"  with the "widget"

**Reply by valinet** [reply]
Marketing-wise you may say, the implementation is kind of different.

**Comment by XuanChen1234** [comment]
I miss this function very much.
![屏幕截图 2023-12-08 181544](https://github.com/valinet/ExplorerPatcher/assets/140480110/79e93dde-2a7c-4b24-9d26-74f4bdbd9a75)

**Reply by valinet** [reply]
For the moment, it’s available in Windows 10.
